setup_chef_server.sh

```diff
@@ -54,6 +63,7 @@ else
   # we can take about 45 minutes to Chef the first machine when running on VMs
   # so follow tuning from CHEF-4253
   printf "erchef['s3_url_ttl'] = 3600\n" >> /etc/chef-server/chef-server.rb
+  export NO_PROXY=${NO_PROXY-127.0.0.1}
```

Comment:
Why set this here and not in [proxy_setup.sh](https://github.com/bloomberg/chef-bach/blob/88b52d57668492b5fb8ab376d1ea3408cf88a86b/proxy_setup.sh#L18)?